src/reducers/index.js

```diff
@@ -18,9 +18,11 @@ const reducer = combineReducers({
   classrooms
 });
 
+const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ && window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__({ trace: true, traceLimit: 25 }) || compose;
```

Comment:
What is this actually doing?

Reply:
its for the redux devtools, so you can see the state changes as they happen.

Reply:
Interesting. Good to know.